pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -5126,6 +5131,233 @@ status:
 	}
 }
 
+func TestReconcile_verifyResolvedV1Task_Success(t *testing.T) {
+	resolverName := "foobar"
+	ts := parse.MustParseV1Task(t, `
+metadata:
+  name: test-task
+  namespace: foo
+spec:
+  params:
+  - default: mydefault
+    name: myarg
+    type: string
+  steps:
+  - script: echo $(inputs.params.myarg)
```

Comment:
shouldn't this be $(params.myarg) - not that it matters for the test itself?

Reply:
oh yes, let me fix this. Yeah it doesn't matter for the test, we just need a task